pkg/delete/delete.go

```diff
@@ -317,16 +320,16 @@ func AllAutomations(ctx context.Context, c automationClient) []error {
 			continue
 		}
 
-		log.Info("Collecting Automations of type %s...", resource)
+		log.FromCtx(ctx).WithFields(loggers.TypeF(string(resource))).Info("Collecting Automations of type %s...", resource)
 		objects, err := c.List(ctx, t)
 		if err != nil {
 			errs = append(errs, err)
 			continue
 		}
 
-		log.Info("Deleting %d Automations of type %s...", len(objects), resource)
+		log.FromCtx(ctx).WithFields(loggers.TypeF(string(resource))).Info("Deleting %d Automations of type %s...", len(objects), resource)
 		for _, o := range objects {
-			log.Debug("Deleting Automation object with id %q...", o.ID)
+			log.FromCtx(ctx).WithFields(loggers.TypeF(string(resource)), loggers.F("object", o)).Debug("Deleting Automation object with id %q...", o.ID)
```

Comment:
objectId maybe?

Reply:
Here and in other cases I've opted to include the full object as a field, rather than just the ID or parts of it :) 

The content of "object" is `o`, so the full object struct, rather than just the ID

Reply:
I'm not 100% sure how helpful that is, but it seemed like an interesting thing to include in the logs. 
I also think that for future simplified debug information sharing we should write structured logs, as they contain so much possibly helpful metadata now!